validate/number/src/index.ts

```
@@ -0,0 +1,165 @@
+        config.sheetSlug || '**',
+        async (record, event: FlatfileEvent) => {
+          for (const field of config.fields) {
+            const numberField = record.get(field)
+
+            try {
+              let numberValue: string | number = String(numberField)
+                .replace(config.thousandsSeparator || ',', '')
+                .replace(config.decimalPoint || '.', '.')
+              numberValue = parseFloat(numberValue)
+
+              if (isNaN(numberValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)